plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -79,6 +131,20 @@ public final void destroy()
         client.close();
     }
 
+    @Test
+    @Override
+    public void testCreateSchema()
+    {
+        throw new SkipException("Schema creation is not supported in the Elasticsearch connector");
```

Comment:
It would be nice to add
```
assertThatThrown(() -> super.testCreateSchema()).
 .hasMessageContaining("....")
```
To validate that reason for original test failure is actually fact that we cannot create schema. 
It will remind us that test need to be changed when (if) we add support for schema creation to ES.